packages/application-generic/src/services/queues/standard-queue.service.ts

```diff
@@ -21,4 +22,18 @@ export class StandardQueueService extends QueueBaseService {
 
     this.createQueue();
   }
+
+  public async addMinimalJob(data: IStandardJobDto) {
+    return await super.addMinimalJob(data);
+  }
```

Comment:
Do we need this overridden function if it's just calling the super? If you want to force the `data` type here I think it's better to do with generics on the `QueueBaseService`. Then we won't need to do everywhere: 
```
public async add(data: T) {
    return await super.add(data);
  }

  public async addBulk(data: T[]) {
    return await super.addBulk(data);
  }
```

Reply:
It will take more time in order to add those methods typed due to the BullMqService.add method restrictions, so for this PR scope I would suggest keeping it like this. wdyt we can pospond it?